vertx-mysql-client/src/main/java/io/vertx/mysqlclient/MySQLConnectOptions.java

```diff
@@ -65,6 +65,7 @@ public static MySQLConnectOptions fromUri(String connectionUri) throws IllegalAr
   public static final Map<String, String> DEFAULT_CONNECTION_ATTRIBUTES;
   public static final SslMode DEFAULT_SSL_MODE = SslMode.DISABLED;
   public static final String DEFAULT_CHARACTER_ENCODING = "UTF-8";
+  public static final int DEFAULT_PIPELINING_LIMIT = 1;
```

Comment:
why not use the same value than PgConnectOptions which is 256 ?

Reply:
I'm assuming this is for backward compatibility ?

Reply:
yes, I'm trying to avoid introducing breaking changes.